Dynamic Programming/Easy/Dice Combinations/Solution.md

```diff
@@ -15,5 +15,5 @@ T(n):
   return T(n-1) + T(n-2) + T(n-3) + T(n-4) + T(n-5) + T(n-6)
 ```
 
-The above solution without memoitation has a time complexity of O(n^6) but reduces to Θ(n*6) when memotized!!!
+The above solution without memoitation has a time complexity of O(n^6) but reduces to Θ(n) when memotized!!!
 So it can be compiled before 1 sec.
```

Comment:
Nice, I also think you have made a small typo: `memoitation -> **memoization**` & `memotized -> **memoized**`

https://en.wikipedia.org/wiki/Memoization

Reply:
Sorry for that !!